lib/elixir/lib/calendar.ex

```diff
@@ -1004,19 +1004,32 @@ defmodule DateTime do
       {:ok, %DateTime{calendar: Calendar.ISO, day: 23, hour: 22, microsecond: {211914, 3}, minute: 53,
                       month: 1, second: 43, std_offset: 0, time_zone: "Etc/UTC", utc_offset: 0,
                       year: 46302, zone_abbr: "UTC"}}
-
+  
+  Negative Unix times are supported, up to -62167219200000000 microseconds, 
```

Comment:
Should the example be with whole seconds (as opposed to microseconds) since whole seconds is the default?